### PR TITLE
Update login page

### PR DESCRIPTION
**Pull request by timcappalli**
Updated login page:
- Added title
- Adding warning about personal data
- Added URL to footer
- Added current version to footer

image

### CODE REVIEW

File: public/login.html [hunk 1]
```diff
@@ -7,18 +7,19 @@
...
     <link rel="stylesheet" href="https://code.getmdl.io/1.3.0/material.indigo-pink.min.css">
     <link rel="stylesheet" href="styles.css">
-
     <link rel="shortcut icon" href="images/favicon.png">
-
-
 </head>
 
 <body id="signInPrompt">
+    <div id="loginHeader">
```
Comment (akshayku) [inline public/login.html:20]: I would get rid of this title. It's more clean the current way.

File: public/login.html [hunk 2]
```diff
@@ -30,21 +31,35 @@ <h2 class="mdl-card__title-text">Sign In</h2>
                     <label class="mdl-textfield__label" for="uid">username</label>
                 </div>
             </form>
+            <p>This site is for dev testing only.<br><span style="color: red; font-weight: bold;">Do not submit any
+                    personal data.</span></p>
```
Comment (akshayku) [inline public/login.html:35]:
```suggestion
            <p>This site is for dev testing WebAuthn interface.<br><span style="color: red; font-weight: bold;">Do not submit any
                    personal data.</span></p>
```

File: public/login.html [hunk 3]
```diff
@@ -30,21 +31,35 @@ <h2 class="mdl-card__title-text">Sign In</h2>
...
         <div class="mdl-card__actions mdl-card--border">
             <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" id="signInButton">
                 Sign In
             </a>
         </div>
     </div>
+    <div id="loginFooter">
+        <p class="text-italic">aka.ms/webauthntest</p>
+        <p id="version" class="text-muted"></p>
+    </div>
```
Comment (akshayku) [inline public/login.html:46]: Can you also add this link to README file?

**Review by akshayku — CHANGES_REQUESTED** [review]

File: public/login.html (hunk 3 above)
Comment (timcappalli) [reply public/login.html:43]: done

File: public/login.html (hunk 2 above)
Comment (timcappalli) [reply public/login.html]: updated

File: public/login.html (hunk 1 above)
Comment (timcappalli) [reply public/login.html]: removed

**Review by akshayku — APPROVED** [review]

**Pull request merged**